### ISSUE TITLE
Use structure and content, not visual contrast, to label past events

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org-11ty#10]
If the intention for reducing contrast on the event cards is to denote that an event has passed (as opposed to disable to prevent access to), content and IA should be planned ahead of time to create space for it (as opposed to a change in the design).

The current visual design technique will afford access to users with high vision in excellent reading conditions (indoors with high screen brightness) to click and view expired events, but will strip access away from users with low vision in less than ideal reading conditions (outdoors in daylight, nighttime with low brightness).
